src/style.css

```diff
@@ -0,0 +1,189 @@
+/* #region Reset */
+:root {
+  box-sizing: border-box;
```

Comment:
it's a good practice to set box-sithing for all elements - `* { box-sizing: border-box; }`